### ISSUE TITLE
Possible to list reward value?

### ISSUE THREAD

**Issue by AlanBarber** [chappjc/votetime#3]
I've just started looking at the app and was thinking it would be handy for tax reporting purposes if it would list the ticket purchase amount and reward value. 

any chance you could add that?

**Comment by chappjc** [comment]
Yes, that would be handy for sure.  I think the most important info for computing tax liability would be vote reward and vote time. Output in CSV for spreadsheet import would be ideal.

However, to say that I'm swamped would be a gargantuan understatement.  This probably won't happen very soon.  PRs are very welcome though!

**Comment by AlanBarber** [comment]
No problem, i'll take a look and see what i can figure out.
